From 0408bf44fec68ca9ef67c861c08b6f8746897e5a Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Tue, 1 Nov 2011 20:05:48 +0100
Subject: [PATCH] =?UTF-8?q?Use=20a=20button-sized=20image=20on=20=E2=80=98Au?=
 =?UTF-8?q?thenticate=E2=80=99=20button?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Jens Georg noticed that the ‘Authenticate’ button used an image that was
very large, causing it to expand too much horizontally. The image was
incorrectly set to be Gtk.IconSize.DIALOG, which should have been
Gtk.IconSize.BUTTON.
---
 vinagre/vinagre-utils.vala |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vinagre/vinagre-utils.vala b/vinagre/vinagre-utils.vala
index da10035..fcc5ea8 100644
--- a/vinagre/vinagre-utils.vala
+++ b/vinagre/vinagre-utils.vala
@@ -121,7 +121,7 @@ namespace Vinagre.Utils {
 
         var ok_button = xml.get_object ("ok_button") as Button;
         var image = new Image.from_stock (Stock.DIALOG_AUTHENTICATION,
-                                          IconSize.DIALOG);
+                                          IconSize.BUTTON);
         ok_button.image = image;
 
         var username_entry = xml.get_object ("username_entry") as Entry;
-- 
1.7.7.2

